charts/rancher-eks-operator/1.0.1000/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.10-alpha1
```

Comment:
I'm not sure questions.yaml actually works, as opposed to questions.yml. But as these are both kev2 operators it doesn't actually matter.